test/suites/common-all/test-maintenance/test-maintenance-mode.ts

```diff
@@ -1,33 +1,44 @@
 import "@tanssi/api-augment";
 
 import { beforeAll, describeSuite, expect } from "@moonwall/cli";
-import type { KeyringPair } from "@moonwall/util";
+import { type KeyringPair, generateKeyringPair } from "@moonwall/util";
 import type { ApiPromise } from "@polkadot/api";
 import { initializeCustomCreateBlock } from "utils";
+import { checkCallIsFiltered } from "helpers";
+import { DANCE } from "utils";
 
 describeSuite({
-    id: "CO0101",
+    id: "C0401",
     title: "Maintenance mode test suite",
     foundationMethods: "dev",
     testCases: ({ it, context }) => {
         let polkadotJs: ApiPromise;
         let alice: KeyringPair;
-        let bob: KeyringPair;
+        let randomAccount: KeyringPair;
         let chain: string;
+        let isStarlight: boolean;
 
         beforeAll(() => {
             initializeCustomCreateBlock(context);
 
             polkadotJs = context.pjsApi;
             chain = polkadotJs.consts.system.version.specName.toString();
             alice = context.keyring.alice;
-            bob = context.keyring.bob;
+            randomAccount = chain === "frontier-template" ? generateKeyringPair() : generateKeyringPair("sr25519");
+            const runtimeName = polkadotJs.runtimeVersion.specName.toString();
+            isStarlight = runtimeName === "starlight";
         });
 
         it({
             id: "E01",
             title: "No maintenance mode at genesis",
             test: async () => {
+                if (isStarlight) {
+                    console.log("Skipping E01 test for Starlight: maintenance mode is not available yet");
+                    expect(polkadotJs.query.maintenanceMode).to.be.undefined;
+                    return;
```

Comment:
I wish there was a way to automatically skip all tests in beforeAll, but there is not